packages/react-conformance/src/defaultTests.tsx

```diff
@@ -254,6 +255,7 @@ export const defaultTests: TestObject = {
       const defaultEl = customMount(<Component {...requiredProps} />);
       const defaultComponent = getComponent(defaultEl, helperComponents, wrapperComponent);
       const defaultClassNames = defaultComponent.getDOMNode().getAttribute('class')?.split(' ') || [];
+      defaultEl.unmount();
```

Comment:
The odd test that calls `mount` more than once. This kind of test cannot be used with a 'global' mount.

I think this is 🤮 but can't think of a better, idea. Suggestions welcome

Reply:
A bit confused, where's the other call? Also not quite understanding what the problem is (but I also don't have a deep understanding of how `mount` works without `attachTo`).

Reply:
I created a `mountWithCleanup` module which is generally how the enzyme community seems to handle the unmounting, however that solution breaks down when `mount` is used more than once in a test.

Unfortunately, this is the only time in all conformance tests this is done. The alternative would be to explicitly unmount in each test.

Reply:
Ah now I see the second `mount` below this one. I'm still not convinced this is a problem given that the test is comparing classNames in two different cases, but theoretically you could do the same thing by mounting the component once and updating props. Or to be certain of avoiding interference/retained state, maybe the test could mount both versions at once inside a fragment?

Reply:
yeah that's a good shout, I'll try to update the test to use a single mount